docs/version0.38.md

```diff
@@ -46,6 +47,12 @@ This option controls the handling of the `SIGUSR2` signal by the VM signal handl
 
 For more information, see [`-XX:[+|-]HandleSIGUSR2`](xxhandlesigusr2.md).
 
+### Technical preview of CRIU feature
+
+A technical preview of CRIU feature is included in this release. It is currently available for OpenJDK 11 and later running on Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only.
```

Comment:
I think we need to mention the nature of CRIU, it's a separate project (also a "tool", according to its creator), not something specific to OpenJ9. Also, this is the first instance of the abbreviation in normal text so this is where it needs to be expanded, not later on. The name for the project on the official website is "Checkpoint/Restore In Userspace" so the expanded name needs modifying anyway. I suggest something like "This release supports, as a technical preview, the Checkpoint/Restore In Userspace (CRIU) tool that is included with many Linux distributions."

Reply:
Done